web/themes/custom/itkdev/itkdev_project_theme/templates/reports-project-track.html.twig

```diff
@@ -82,9 +82,28 @@
           </div>
           <div class="my-3 flex flex-wrap px-4">
             <div class="basis-full mb-5">
-                <div> {{ dimensions[2] is not empty ? '@dimension is illustrated by bubble size'|t({'@dimension': dimensions[2]}) }}</div>
+              <div>{{ dimensions[2] is not empty ? '@dimension is illustrated by bubble size'|t({'@dimension': dimensions[2]}) }}</div>
             </div>
           </div>
+          {% if trackTypeConfig.quadrant_texts is not empty and projectCount is same as 1 %}
+            {% set activeQuadrant = track.entity.getConfiguration.bubbleChartReportResult.activeQuadrant %}
+            <div>
+              {% if activeQuadrant is same as '2' or activeQuadrant is same as '4' %}
```

Comment:
Consider using `==` rather than `is same as` here. Using `is same as` also checks types, but you risk that PHP or a Yaml parser (suddenly) read `'2'` and `2` (the integer) and then these checks will fail.